data/resume/jbatorski.json

```diff
@@ -0,0 +1,182 @@
+{
+  "$schema": "https://raw.githubusercontent.com/jsonresume/resume-schema/v1.0.0/schema.json",
+  "basics": {
+    "name": "Jerry Batorski",
+    "label": "Frontend Software Engineer",
+    "email": "jerry.batorski@gmail.com",
+    "url": "https://jarekb84.github.io/me/",
+    "summaryOfQualifications": [
```

Comment:
Summary is a property on the [jsonresume schema](https://github.com/jsonresume/resume-schema/blob/master/schema.json#L46), but it can only be a string, and I like to have a bullet point list here instead. In the future, I may opt for a paragraph summary since that may read better.


`summaryOfQualifications` is not defined on the jsonresume schema, but it does have [additionalProperties](https://github.com/jsonresume/resume-schema/blob/master/schema.json#L19) set to true on the basics object. Looking at the json-schma.org [docs](http://json-schema.org/understanding-json-schema/reference/object.html#additional-properties), seems this can be used to extend the schema with custom properties. I opted for this approach to let me consolidate multiple jobs under one entry, mainly as a way to reduce content which isn't relevant and not take up lots of reading space with job breakdowns. 

The tradeoff here is that this schema and its content wouldn't be usable by jsonresume's [theming capabilities](https://jsonresume.org/themes/), but I went ahead with this to allow flexibility in visual representation. If I abandon this react based github approach, switching back to standard based properties will be straight forward.